NCP/openncp-configuration/tm.properties

```diff
@@ -1,34 +1,32 @@
-###########################################################################################
-#                                                                              		  #
-#                                                                              		  #
-# TM configuration                                                           		  #
-#                                                                              	  	  #
-#                                                                              	          #
-# This is the CI environment version.                                          		  #
+��������###########################################################################################
```

Comment:
I don't know where these `²`'s are coming from, but I think we better remove them.